src/main/java/io/github/cottonmc/resources/block/DefaultBlocks.java

```diff
@@ -1,22 +1,68 @@
 package io.github.cottonmc.resources.block;
 
+import java.util.Map;
+import java.util.function.Supplier;
+
+import com.google.common.collect.ImmutableMap;
+
 import io.github.cottonmc.cotton.registry.CommonBlocks;
-import net.minecraft.block.*;
-import net.minecraft.block.Block.*;
+import net.fabricmc.fabric.block.FabricBlockSettings;
+import net.fabricmc.fabric.tags.FabricItemTags;
+import net.minecraft.block.Block;
+import net.minecraft.block.Material;
+import net.minecraft.block.OreBlock;
 
 public class DefaultBlocks {
+    /** A generic metal ore supplier, similar to the properties of coal or iron ore. */
+    private static final Supplier<Block> STONE_TIER_ORE = ()->new OreBlock(FabricBlockSettings.of(Material.STONE)
```

Comment:
I would move the suppliers in another class and only have the map and the method that actually creates a block in this class.